src/components/receipt-view/utils/calculations.js

```diff
@@ -0,0 +1,141 @@
+// Data
```

Comment:
Everything in this file looks like it would be _super easy_ to test (and being focused on business logic, probably is really valuable to test too). Any chance you all will have time to get to some of that stuff, too?

Reply:
I think the chances are low, but worth looking in to if we find a little more time in the next week or so.